lib/checks/keyboard/focusable-disabled-evaluate.js

```diff
@@ -29,7 +30,19 @@ function focusableDisabledEvaluate(node, options, virtualNode) {
   if (relatedNodes.length === 0 || isModalOpen()) {
     return true;
   }
-  return relatedNodes.every(related => related.onfocus) ? undefined : false
+  return relatedNodes.every(related => {
+    const vNode = getNodeFromTree(related);
```

Comment:
I think a refactor is in order to avoid this lookup. We had the virtual nodes of these 15 lines higher.